### DISCUSSION TITLE
Προθεσμία τελευταίου παραδοτέου

### DISCUSSION THREAD

**Discussion by vxsilis** [courses-ionio/sw#941] [Announcements]
Καλησπέρα σας,

Θα ήθελα να ρωτήσω πότε είναι η προθεσμία του τελευταίου (12ου) παραδοτέου στο μάθημα. Την πρώτη βδομάδα που ανοίγουν οι σχολές μετά το πάσχα;

Ευχαριστώ

**Comment by epidrome** [comment]
όπως λένε οι οδηγίες 
https://github.com/courses-ionio/sw

> *Η συμπλήρωση της εβδομάδας ορίζεται ως η μέρα πριν το επόμενο μάθημα, π.χ., για την 1η εβδομάδα η παράδοση είναι την ημέρα πριν το 2ο μάθημα.

επομένως, το 12ο παραδοτέο αντιστοιχεί στην 13η εβδομάδα, που είναι η τελευταία εβδομάδα μαθημάτων
https://di.ionio.gr/gr/news/academic-calendar/
